lib/Dialect/TorchConversion/Transforms/VerifyMhloBackendContract.cpp

```diff
@@ -45,8 +46,8 @@ class VerifyMhloBackendContractPass
     ConversionTarget target(*context);
 
     // Structural operations.
-    target.addDynamicallyLegalOp<ModuleOp, func::FuncOp, func::ReturnOp>(
-        opHasLegalTypes);
+    target.addDynamicallyLegalOp<ModuleOp, func::FuncOp, func::ReturnOp,
```

Comment:
nit: Structural operations comment does not seem correct for ShapeOfOp -- recommand adding it in a new call to `target.addDynamicallyLegalOp` for clarity.

Reply:
done